packages/amplify_datastore/lib/amplify_datastore.dart

```diff
@@ -154,6 +149,7 @@ class AmplifyDataStore extends DataStorePluginInterface {
           Map<String, String>.from(e.details as Map),
         );
       } else if (e.code == 'AmplifyAlreadyConfiguredException') {
+        logger.info('Ignoring Amplify Native already configured exception');
```

Comment:
I would make all logs debug unless there's a good reason customers should be aware of them.

```suggestion
        logger.debug('Ignoring Amplify Native already configured exception');
```